logs_rollbar.admin.inc

```diff
@@ -52,6 +52,14 @@ function logs_rollbar_admin_settings($form, &$form_state) {
     '#default_value' => variable_get('logs_rollbar_auto_disable_on_local', TRUE),
   );
 
+  $form['logs_rollbar_ignored_clientside_messages'] = array(
+    '#type' => 'textfield',
```

Comment:
Better to use [textarea](https://api.drupal.org/api/drupal/developer%21topics%21forms_api_reference.html/7.x#textarea) simce it might be very long list.